src/api/contract/getWithdrawalBalance.js

```diff
@@ -1,14 +1,21 @@
 const DVFError = require('../../lib/dvf/DVFError')
 const BN = require('bignumber.js')
 
-module.exports = async (dvf, token) => {
-  const starkKey = await dvf.contract.getStarkKey()
+module.exports = async (dvf, token, starkKey) => {
+  const starkKey = dvf.config.starkExUseV2
```

Comment:
I'm getting an error here trying to initialise DVF
`SyntaxError: Identifier 'starkKey' has already been declared`